gensim/models/fasttext.py

```diff
@@ -793,7 +807,6 @@ def _load_model_params(self, file_handle):
             ws = version
             epoch, min_count, neg, _, loss, model, bucket, minn, maxn, _, t = self.struct_unpack(file_handle, '@10i1d')
         # Parameters stored by [Args::save](https://github.com/facebookresearch/fastText/blob/master/src/args.cc)
-        self.wv.vector_size = dim
```

Comment:
Because self.wv gets initialized to default values.

The current FB model loading code uses something that feels like an antipattern. It creates a model with default parameters, and then overrides those parameters one by one.

Reply:
> I think a better alternative would be to pass the final values of the parameters to the model consructor.

I agree, OK